### PR TITLE
Adding version.Info in apiserver.go

### LINKED ISSUE #17176: Version API in swagger spec return type is void
https://github.com/kubernetes/kubernetes/blob/master/api/swagger-spec/version.json#L12

Why is this void? Makes no sense.

### PR DESCRIPTION
**Pull request by Pensu**
Fixes #17176

### CODE REVIEW

**Comment by luxas** [comment]
ok to test

**Comment by lavalamp** [comment]
LGTM, but you'll need to run the doc generation tools (hack/update-all.sh)

**Comment by Pensu** [comment]
@lavalamp Ok, sure, so should I run doc generation tool and send a new PR?

**Comment by lavalamp** [comment]
@Pensu just push to your branch again, the pr will be updated.

**Comment by luxas** [comment]
@Pensu Here are some steps you may take to update:
```
# Change directory to the source
cd kubernetes
# See that you're on master
git branch
# You should see your commit here
git log -n 1
# Go from a commit to staging, in order to add more files
git reset --soft HEAD~
# Update Kubernetes files
hack/update-all.sh
# See which files changed and add them
git status
git add ...
# Commit your changes
git commit -m "Your original commit message here"
# ... and push the branch
git push -f origin master
```

**Comment by Pensu** [comment]
@lavalamp Done.
@luxas Thanks for the steps :+1:

**Comment by lavalamp** [comment]
This sort of makes a non-api type into an official part of our API. I think I'm OK with this in this case? But it shows a bunch of fields as required and I think they'd be better optional. We could add omitempty tags or just live with it as-is.

I think I'll LGTM as-is but tag @bgrant0607 in case he disagrees.

**Comment by lavalamp** [comment]
ok to test

**Comment by bgrant0607** [comment]
@lavalamp Is there a reason we shouldn't move that to pkg/api/version?

**Comment by bgrant0607** [comment]
@lavalamp Or to pkg/api/unversioned?

**Comment by lavalamp** [comment]
pkg/api/unversioned would be the place. The reason not to do it is that it'd be very invasive at the moment.

**Comment by luxas** [comment]
@k8s-bot unit test this please github issue: #IGNORE

**Comment by luxas** [comment]
@Pensu unit/integration fails with:
```
Verifying ./hack/../hack/verify-gofmt.sh
!!! 'gofmt -s' needs to be run on the following files: 
./pkg/apiserver/apiserver.go
FAILED   ./hack/../hack/verify-gofmt.sh	9s
```

**Comment by luxas** [comment]
@Pensu Please squash to just one commit before lgtm

**Comment by lavalamp** [comment]
Please squash, thanks!

**Comment by Pensu** [comment]
@luxas @lavalamp Done.

**Comment by Pensu** [comment]
@luxas @lavalamp Done.

File: pkg/apiserver/apiserver.go [hunk 1]
```diff
@@ -174,7 +174,8 @@ func InstallVersionHandler(mux Mux, container *restful.Container) {
 			Doc("get the code version").
 			Operation("getCodeVersion").
 			Produces(restful.MIME_JSON).
-			Consumes(restful.MIME_JSON))
+			Consumes(restful.MIME_JSON).
+			Writes(version.Info{}))
```
Comment (dims) [inline pkg/apiserver/apiserver.go]: Do we really need this change? with the latest code in  curl -kvs http://127.0.0.1:8080/version already prints the version numbers

```
$ curl -kvs http://127.0.0.1:8080/version
*   Trying 127.0.0.1...
* Connected to 127.0.0.1 (127.0.0.1) port 8080 (#0)
> GET /version HTTP/1.1
> Host: 127.0.0.1:8080
> User-Agent: curl/7.47.0
> Accept: */*
>
< HTTP/1.1 200 OK
< Content-Type: application/json
< Date: Mon, 06 Jun 2016 13:21:42 GMT
< Content-Length: 295
<
{
  "major": "1",
  "minor": "3+",
  "gitVersion": "v1.3.0-alpha.4.1106+68097515d82121-dirty",
  "gitCommit": "68097515d821217918a1c916759a8373e5e5eb86",
  "gitTreeState": "dirty",
  "buildDate": "2016-06-06T13:18:29Z",
  "goVersion": "go1.6.1",
  "compiler": "gc",
  "platform": "linux/amd64"
* Connection #0 to host 127.0.0.1 left intact
}
```

Comment (Pensu) [inline pkg/apiserver/apiserver.go]: Swagger spec still returns void. Probably @nikhiljindal can comment.

Comment (nikhiljindal) [inline pkg/apiserver/apiserver.go]: swagger-spec is auto generated from this code.
@dims You are right that the response right now is correct. But our documentation of response (in swagger spec) is incorrect. This change just fixes that.

**Comment by nikhiljindal** [comment]
LGTM, thanks!

File: pkg/apiserver/apiserver.go (hunk 1 above)
Comment (dims) [inline pkg/apiserver/apiserver.go]: AH!! my bad thanks @nikhiljindal i missed that.

**Pull request merged**